example/ios/example.xcodeproj/project.pbxproj

```diff
@@ -942,6 +630,7 @@
 				COPY_PHASE_STRIP = YES;
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
```

Comment:
This shouldn't be needed if using XCFrameworks as we are in our podspec. Does it work if you remove this excluded arch setting?

Reply:
I've removed this change but this gets added automatically even after I remove it when I run the app after a fresh pod install.